.github/workflows/release.yml

```diff
@@ -0,0 +1,16 @@
+name: Release-Pipeline
+
+on:
+  workflow_run:
+    workflows: [CI-Pipeline]
+    branches: [main]
+    types:
+      - completed
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  call-terraform-release-pipeline:
+    uses: terraform-ibm-modules/common-dev-assets/.github/workflows/release.yml@main
```

Comment:
The release needs to point to common-pipeline-assets (see above), and also will require a conditional statement so that release will only trigger if CI was successful. You can copy the file we have in the template: https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blob/main/.github/workflows/release.yml